ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -113,6 +138,11 @@ open class CommandBar: UIView {
         updateShadow()
     }
 
+    // MARK: - TokenizedControl
+
+    public typealias TokenSetKeyType = CommandBarTokenSet.Tokens
+    public var tokenSet: CommandBarTokenSet
```

Comment:
curious what's the archtecture difference here which some control tokenSet is a "lazy" public var and this one isn't? @sophialee0416 & @mischreiber

Reply:
I think it's probably be cause the other controls needed to pass in a style or something to their token set, so they needed to be lazy. Since the `CommandBarTokenSet` doesn't rely on anything from the control, it can just be initialized.